examples/transform_speed.ipynb

```diff
@@ -0,0 +1,365 @@
+      "Numpy version: 1.18.1\n",
+      "Pytorch version: 1.4.0\n",
+      "Ignite version: 0.3.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import os\n",
+    "import sys\n",
+    "from glob import glob\n",
+    "import tempfile\n",
+    "\n",
+    "import numpy as np\n",
+    "import nibabel as nib\n",
+    "\n",
+    "\n",
+    "import torch\n",
+    "from torch.utils.data import DataLoader\n",
+    "from torch.multiprocessing import Pool, Process, set_start_method\n",
+    "try:\n",
+    "     set_start_method('spawn')\n",
+    "except RuntimeError:\n",
+    "    pass\n",
+    "\n",
+    "sys.path.append('..') # assumes this is where MONAI is\n",
+    "\n",
+    "import monai\n",
+    "from monai.transforms.compose import Compose\n",
+    "from monai.data.nifti_reader import NiftiDataset\n",
+    "from monai.transforms import (AddChannel, Rescale, ToTensor, \n",
+    "                              UniformRandomPatch, Rotate, RandAffine)\n",
+    "\n",
+    "monai.config.print_config()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 0. Preparing input data (nifti images)\n",
+    "\n",
+    "Create a number of test Nifti files, 3d single channel images with spatial size (256, 256, 256) voxels."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tempdir = tempfile.mkdtemp()\n",
+    "\n",
+    "for i in range(5):\n",
+    "    im, seg = monai.data.synthetic.create_test_image_3d(256,256,256)\n",
+    "    \n",
+    "    n = nib.Nifti1Image(im, np.eye(4))\n",
+    "    nib.save(n, os.path.join(tempdir, 'im%i.nii.gz'%i))\n",
+    "    \n",
+    "    n = nib.Nifti1Image(seg, np.eye(4))\n",
+    "    nib.save(n, os.path.join(tempdir, 'seg%i.nii.gz'%i))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# prepare list of image names and segmentation names\n",
+    "images = sorted(glob(os.path.join(tempdir,'im*.nii.gz')))\n",
+    "segs = sorted(glob(os.path.join(tempdir,'seg*.nii.gz')))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Test image loading with minimal preprocessing"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "torch.Size([3, 1, 256, 256, 256]) torch.Size([3, 1, 256, 256, 256])\n"
+     ]
+    }
+   ],
+   "source": [
+    "imtrans = Compose([\n",
+    "    AddChannel(),\n",
+    "    ToTensor()\n",
+    "])    \n",
+    "\n",
+    "segtrans = Compose([\n",
+    "    AddChannel(),\n",
+    "    ToTensor()\n",
+    "])    \n",
+    "    \n",
+    "ds = NiftiDataset(images, segs, transform=imtrans, seg_transform=segtrans)\n",
+    "loader = DataLoader(ds, batch_size=3, num_workers=8)\n",
```

Comment:
Why num_workers is 8 instead of 4 here? It's different from your other CPU test.
Thanks.